metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -0,0 +1,71 @@
+from os import listdir
+from os.path import isfile, join
+from subprocess import run, PIPE
+from typing import List
+
+from .... import R
+
+import kfp
+
+import pytest
+
+"""
+From the folder `tests`, run: 
+`python -m pytest -s -n 3 run_integration_tests.py`
+
+This script runs all the flows in the `sample_flows` directory. It creates
+each kfp run, waits for the run to fully complete, and prints whether
+or not the run was successful. It also checks to make sure the logging
+functionality works.
+
+More specifically, the tests spawn KFP runs and ensure the spawning processes
+have a returncode of 0. However, the logs of the runs within KFP are simply logged
+to the screen, and it is the responsibility of the user to read these logs to ensure
+the run executed correctly on KFP.
+
+Parameters:
+-n: specifies the number of parallel processes used by PyTest.
+
+Sometimes, the tests may fail on KFP due to resource quota issues. If they do,
+try reducing -n (number of parallel processes) so less simulateneous
+KFP runs will be scheduled.
+
+"""
+
+
+def _python():
+    if R.use_r():
+        return "python3"
```

Comment:
these tests aren't going to use R for the foreseeable future?

Reply:
@talebzeghmi this is how Metaflow uses the current Python executable. See [here](https://github.com/zillow/metaflow/blob/4a4c4f3473a15fbb92017700a68eabeffaf048db/metaflow/plugins/kfp/kfp.py#L275).